tools/deb/test/run_gce_test.sh

```diff
@@ -304,8 +313,16 @@ function setUp() {
  LOCAL_IP=$(istioVMInternalIP $TESTVM)
 
  # Configure a service for the local nginx server, and add an ingress route
- istioConfigHttpService rawvm 80 $LOCAL_IP
- istioRoute "/${TESTVM}/" rawvm 80
+ istioConfigHttpService $TESTVM 80 $LOCAL_IP
+
+ # Bookinfo components on the VM
+ istioConfigService ${TESTVM}-details 9080 $LOCAL_IP
+ istioConfigService ${TESTVM}-productpage 9081 $LOCAL_IP
+ istioConfigService ${TESTVM}-ratings 9082 $LOCAL_IP
+ istioConfigService ${TESTVM}-mysql 3306 $LOCAL_IP
+
+ # Prepare simple test to make sure istio-ingress can reach a http server on the VM
+ istioRoute "/${TESTVM}/" ${TESTVM} 80
```

Comment:
Why do we need this elaborate setup for this simple test? Isn't it enough to run a single nginx service with echo modules to emulate multiple services ? If there is no label based routing test, you just need to test http and tcp.

Reply:
Already have the nginx service and echo, for the simple test. 

The script is intended to setup an environment for end-to-end tests, including testing
the bookinfo works on VM. Eventually will run load tests and maybe long-running
stability tests. We basically need to have similar coverage for VM as we have 
for K8S - the setup will be amortized. 

( also it's easier to have a script to setup everything rather than remembering all commands)